src/components/page/blocks/ReferenceBlock.jsx

```diff
@@ -27,6 +27,8 @@ function ReferenceBlock({ block, blockShared, row }) {
   } = block;
 
   const { position } = options || 'left';
+  const defaultIcon = 'lightbulb';
+  const iconButton = button.icon && !button.text;
```

Comment:
`iconButton` is buttons which are _only_ icons, as opposed to a plain text button or a button with text and an icon.